generated/generics/generate.sh

```diff
@@ -32,11 +32,16 @@ fi
 # [1]: https://github.com/cheekybits/genny
 
 mkdir -p $GENERATED_PATH/postingsgen
-cat $GENERIC_MAP_IMPL                                                    \
-  | genny -out=${GENERATED_PATH}/postingsgen/generated_map.go            \
-    -pkg=postingsgen gen "KeyType=[]byte ValueType=postings.MutableList"
+cat $GENERIC_MAP_IMPL                                                  \
+| genny -out=${GENERATED_PATH}/postingsgen/generated_map.go            \
```

Comment:
While you're here, could you change the name to `.../postingsgen/map_gen.go` (for this one and others below) . `_gen.go` is the convention Rob started using in m3x/m3db recently. 

And also update `.excludecoverage`